react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -210,5 +214,6 @@ if (isJitPack) {
         backupHermesReleaseAAR.dependsOn(copyJSBundle)
         copyJSBundle.dependsOn(buildJSBundle)
         buildJSBundle.dependsOn(yarn_install, ensureAssetsDirectory)
+        ensureAssetsDirectory.dependsOn(removeReactNativeAztecOldSubmodule)
```

Comment:
👋 @ceyhun , deleting during the gradle build should do the trick as well, but I was thinking that perhaps we don't make this part of the build process and instead do the cleaning in the JitPack config file, [here for example](https://github.com/wordpress-mobile/gutenberg-mobile/blob/6352c210a79dbcf7e9285971761e8eaf94100a01/jitpack.yml#L2). The "cleanup" is mainly a git checkout issue, not a build issue so, I'd expect it not to be involved in the gradle steps. WDYT?

Reply:
Yeah that might be better. Let me try that too 👍